env-var-docs/update-container-parser-package

```diff
@@ -0,0 +1,28 @@
+#! /bin/sh
+
+# DOC: This script is for running inside the parser-package container, which is why it
+# is not located in the civiform/bin directory. It checks if the parser-package has
+# any changes since the package was last installed in the container. If so, reinstall
+# the package.
+
+# Install PyGithub if it isn't installed yet
+# Using grep -q would cause a broken pipe error since it exits as soon
+# as it finds what it needs, so send the output to /dev/null instead
+pip list | grep PyGithub >/dev/null
+if [ "$?" -gt 0 ]; then
```

Comment:
**nitpick:** see https://google.github.io/styleguide/shellguide.html#s6.3-tests

Reply:
Oh woops, yeah would prefer we use bash for consistency. `/bin/sh` is aliased to `/bin/busybox` which runs the `ash` shell and I don't think most of us are up on what we can use in `ash` vs `bash`, so would prefer we just stick with the latter.